.github/workflows/build-abi.yml

```diff
@@ -44,4 +44,4 @@ jobs:
       # Abigail ABI checker
       - name: Check ABI compatibility
         run: |
-          abidiff abi-compat/jammy_0.107.xml _build/src/libnetplan.so.0.0 --headers-dir2 include/ --header-file2 src/abi.h --suppressions abi-compat/suppressions.abignore --no-added-syms
+          abidiff abi-compat/jammy_1.0.xml _build/src/libnetplan.so.1 --headers-dir2 include/ --header-file2 src/abi.h --suppressions abi-compat/suppressions.abignore --no-added-syms
```

Comment:
question: do we have plans to update jammy to netplan 1.0? I'm just confused with the name of the file...